src/__tests__/components/Modal/Modal.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { configure, mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+import Adapter from 'enzyme-adapter-react-15';
+
+import Modal from '../../../components/Modal/Modal';
+
+configure({ adapter: new Adapter() });
+
+describe('Modal component', () => {
+    const render = props => mount(<Modal {...props} />);
```

Comment:
You can probably pass the `requiredProps` directly in this render function, so you don't have to the following over and over again:

```javascript
        const props = {
             ...requiredProps,
             title: 'Modal test title',
         };
```

Reply:
We will also need to pass it again in `runCommonTest` if we pass it on the render function

I think redefining the props on every test makes it easier to debug since you don't need to search around to find the contents of `props` 🙂